src/sentry/services/hybrid_cloud/app/impl.py

```diff
@@ -126,12 +126,15 @@ def get_related_sentry_app_components(
         type: str,
         group_by: str = "sentry_app_id",
     ) -> Mapping[str, Any]:
-        return SentryAppInstallation.objects.get_related_sentry_app_components(
-            organization_ids=organization_ids,
-            sentry_app_ids=sentry_app_ids,
-            type=type,
-            group_by=group_by,
-        )
+        return {
+            str(k): v
```

Comment:
Are there cases where `k` will be `None` (resulting in `"None"` instead of `None`)?